src/test/java/api/loans/DeclareClaimedReturnedItemAsMissingApiTests.java

```diff
@@ -104,11 +115,30 @@ public void cannotDeclareItemMissingWhenLoanIsNotFound() {
       .attemptDeclareClaimedReturnedItemAsMissing(
         new DeclareClaimedReturnedItemAsMissingRequestBuilder()
           .forLoan(notExistentLoanId)
-          .withComment("testing"));
+          .withComment(TESTING_COMMENT));
 
     assertThat(response.getStatusCode(), is(404));
   }
 
+  @Test
+  public void shouldCreateNoteWhenItemDeclaredMissing() {
+    assertThat(notesClient.getAll().size(), is(0));
+
+    claimItemReturnedFixture.claimItemReturned(new ClaimItemReturnedRequestBuilder()
+      .forLoan(loanId)
+      .withItemClaimedReturnedDate(DateTime.now()));
+
+    claimItemReturnedFixture.declareClaimedReturnedItemAsMissing(
+      new DeclareClaimedReturnedItemAsMissingRequestBuilder()
+        .forLoan(loanId)
+        .withComment(TESTING_COMMENT));
+
+    List<JsonObject> notes = notesClient.getAll();
+    assertThat(notes.size(), is(1));
+    assertThat(notes.get(0).getString("title"), is("Claimed returned item marked lost"));
+    assertThat(notes.get(0).getString("domain"), is("loans"));
```

Comment:
I think it is fine to extend the existing 'success' test `canDeclareItemMissingWhenClaimedReturned` with the checks that the note is created.